sympy/solvers/solveset.py

```diff
@@ -69,10 +69,6 @@ class NonlinearError(ValueError):
     """Raised when unexpectedly encountering nonlinear equations"""
     pass
 
-
-_rc = Dummy("R", real=True), Dummy("C", complex=True)
-
-
 def _masked(f, *atoms):
```

Comment:
There should be two blank lines between top-level functions. Notice the spacing in the rest of the file.